app/src/main/java/com/example/pinkorange/vibrato/LiveWithSettings.java

```diff
@@ -256,12 +280,12 @@ protected void onDestroy() {
     }
 
     private void setLoudnessSeekBar() {
+        loudness.setProgress(50);
```

Comment:
Nice! Just a note: in the future, I think we can pass the default setting in as a variable to `setLoudnessSeekBar` and `setBassSeekBar`. If we create a preferences page in the future where users can tell us about their degree and ranges of Hearing Loss, we can use that information to create a customized default setting.